content/en/docs/concepts/scheduling-eviction/scheduling-framework.md

```diff
@@ -129,17 +129,23 @@ Plugins wishing to perform "pre-reserve" work should use the
 NormalizeScore extension point.
 {{< /note >}}
 
-### Reserve
-
-This is an informational extension point. Plugins which maintain runtime state
-(aka "stateful plugins") should use this extension point to be notified by the
-scheduler when resources on a node are being reserved for a given Pod. This
-happens before the scheduler actually binds the Pod to the Node, and it exists
-to prevent race conditions while the scheduler waits for the bind to succeed.
-
-This is the last step in a scheduling cycle. Once a Pod is in the reserved
-state, it will either trigger [Unreserve](#unreserve) plugins (on failure) or
-[PostBind](#post-bind) plugins (on success) at the end of the binding cycle.
+### Reserve {#reserve}
+
+The reserve extension point happens before the scheduler actually binds a Pod to
+a node and its uses are three-fold. First, it serves as a hook to allocate
+resources in the designated node that the scheduled Pod might need.  Second, it
```

Comment:
It shouldn't be used to allocate resources. Such actions are usually costly and should be done in PreBind.

Reply:
We should also preserve the phrasing "Plugins which maintain runtime state (aka "stateful plugins") should use this extension point"

Reply:
Is the volume binding plugin an exception then? I'm now confused as to whether reserve plugins are purely informational or if they can have side effects other than cycle state changes.

Reply:
Volume binding plugin only updates its cache. It doesn't bind the volumes. That's done in PreBind.